packages/slate/src/interfaces/path.ts

```diff
@@ -386,12 +386,14 @@ export const Path = {
             const copy = onp.slice()
 
             if (Path.endsBefore(op, onp) && op.length < onp.length) {
-              const i = Math.min(onp.length, op.length) - 1
-              copy[i] -= 1
+              copy[op.length - 1] -= 1
```

Comment:
Note: This is just clean up. The length check in the if already guaranteed which path is less.